scalableminds/webknossos
# Fix brush performance for coarse mags & avoid some unnecessary re-renders

### URL of deployed dev instance (used for testing):
- https://fixbrushperf.webknossos.xyz

### Steps to test:
- brush in a dataset at mag 512 (needs resolution restriction during setup) --> performance should be similar to brushing in mag 1
  - see here for a good testing DS: https://scm.slack.com/archives/C3RUHRXBM/p1671526852728559
- activate react devtools and check that the toolbar doesn't re-render constantly on mouse move

### Issues:
- perf perf perf

------
(Please delete unneeded items, merge only when none are left open)
